gossipsub.go

```diff
@@ -1,22 +1,16 @@
 package pubsub
 
 import (
-	"context"
 	"fmt"
 	"math/rand"
 	"time"
 
 	pb "github.com/libp2p/go-libp2p-pubsub/pb"
 
-	host "github.com/libp2p/go-libp2p-host"
 	peer "github.com/libp2p/go-libp2p-peer"
 	protocol "github.com/libp2p/go-libp2p-protocol"
 )
 
-const (
-	GossipSubID = protocol.ID("/meshsub/1.0.0")
```

Comment:
don't remove this from here.

Reply:
This was moved to a new file because I wanted to separate a particular gossipsub implementation from configurable gossipsub. However, I don't feel strongly as to where it lives. If you really think they should all be in the same file that's fine.

Reply:
yes, it should live here.

Reply:
ok